Resources/Prototypes/_NF/Loadouts/mercenary_loadout_groups.yml

```diff
@@ -217,46 +219,6 @@
   - ContractorClothingBackpackDuffelChemistry
   - ContractorClothingBackpackSatchelChemistry
   - ContractorClothingBackpackMessengerChemistry
-  - MercenaryClothingBackpackMercenary
-  - MercenaryClothingBackpackDuffelMercenary
-  - MercenaryClothingBackpackSatchelMercenary
-  - MercenaryClothingBackpackMessengerMercenary
-  - ContractorClothingBackpack
-  - ContractorClothingBackpackDuffel
-  - ContractorClothingBackpackSatchel
-  - ContractorClothingBackpackMessenger
-  - PilotClothingBackpackPilot
-  - PilotClothingBackpackDuffelPilot
-  - PilotClothingBackpackSatchelPilot
-  - PilotClothingBackpackMessengerPilot
-  - ContractorClothingBackpackCaptain
-  - ContractorClothingBackpackDuffelCaptain
-  - ContractorClothingBackpackSatchelCaptain
-  - ContractorClothingBackpackMessengerCaptain
-  - ContractorClothingBackpackSalvage
-  - ContractorClothingBackpackDuffelSalvage
-  - ContractorClothingBackpackSatchelSalvage
-  - ContractorClothingBackpackMessengerSalvage
-  - ContractorClothingBackpackCargo
-  - ContractorClothingBackpackDuffelCargo
-  - ContractorClothingBackpackSatchelCargo
-  - ContractorClothingBackpackMessengerCargo
-  - ContractorClothingBackpackHydroponics
-  - ContractorClothingBackpackDuffelHydroponics
-  - ContractorClothingBackpackSatchelHydroponics
-  - ContractorClothingBackpackMessengerHydroponics
-  - ContractorClothingBackpackScience
-  - ContractorClothingBackpackDuffelScience
-  - ContractorClothingBackpackSatchelScience
-  - ContractorClothingBackpackMessengerScience
-  - ContractorClothingBackpackMedical
-  - ContractorClothingBackpackDuffelMedical
-  - ContractorClothingBackpackSatchelMedical
-  - ContractorClothingBackpackMessengerMedical
-  - ContractorClothingBackpackChemistry
-  - ContractorClothingBackpackDuffelChemistry
-  - ContractorClothingBackpackSatchelChemistry
-  - ContractorClothingBackpackMessengerChemistry
```

Comment:
wait why the backpack removal

Reply:
There's a duplicated set.  Not a full set.  If you search before the PR is in, you'll have two instances of most items.

Reply:
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/9fa34463-a556-457a-8af7-a2ea5810c17f)
Screenshot of the merc backpack loadout selection as of `659c11e1b47`.